tests/ltp/Makefile

```diff
@@ -51,19 +52,28 @@ run-sw: $(ROOT_FS)
 	@${LTP_TEST_SCRIPT} run-sw
 
 run-hw-single: $(ROOT_FS)
-	${SGXLKL_STARTER} --hw-debug $(ROOT_FS) $(test)
+	@rm -fr $(ROOT_FS_FRESH_COPY)
```

Comment:
Why specify `-r`? This is not a directory.

Reply:
Removed.